README.md

```diff
@@ -59,6 +70,15 @@ kubectl delete namespace cattle-resources-system
 
 ----
 
+## More Info
+
+Our default chart is built for the use-case of backing up and restoring the Rancher application.
+However, under the hood the Backup Restore Operator is rather flexible extension for backup and restore of Kubernetes resources.
+
+* This operator provides ability to backup and restore Kubernetes applications (metadata) running on any cluster. It accepts a list of resources that need to be backed up for a particular application. It then gathers these resources by querying the Kubernetes API server, packages all the resources to create a tarball file and pushes it to the configured backup storage location. Since it gathers resources by querying the API server, it can back up applications from any type of Kubernetes cluster.
```

Comment:
"provides the ability"

Reply:
I'm not sure what you mean "for a particular application". Isn't Rancher the the only application it backs up?

Reply:
Also, I would insert an Oxford comma like so: `packages all the resources to create a tarball file, and pushes it to`. -- that's in the Microsoft style guide, so it's ok here

Reply:
> I'm not sure what you mean "for a particular application". Isn't Rancher the the only application it backs up?

Adjusting that to read as "backed up for the application" - since by default "the application" is Rancher. But this section is highlighting that with care and caution that BRO can backup other application data too. Technically I didn't write the wording here, just reorganizing it to a new section. But I like these improvements too!